### PR TITLE
Consolidate, correct and test devscripts checks

### PR DESCRIPTION
**Pull request by cjeanner**
With the big changes introduced by #1777, some of the checks done in
devscripts to ensure the OCP cluster was present were broken or not
properly scaling.

This patch corrects this situation, and ensures the tests are properly
working with a dedicated molecule scenario.

It also makes a small logical correction in the reproducer role, to
ensure we don't do same tasks multiple times for nothing, and also to
ensure we're testing the right, up-to-date content.

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running

### CODE REVIEW

**Review by cescgina — APPROVED** [review]
code wise lgtm, but have little context for the changes. What jobs could this affect, the VA ones?

File: roles/devscripts/tasks/get_ocp_vips.yml
Comment (hjensas) [inline roles/devscripts/tasks/get_ocp_vips.yml:1]: \o/ we got rid of this.

**Comment by cjeanner** [comment]
Anything using devscripts - so mostly VA, yes.

**Review by hjensas — APPROVED** [review]

**Pull request merged**